src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/ReportedAdjudicationDto.kt

```diff
@@ -164,3 +168,15 @@ data class HearingSummaryDto(
   @Schema(description = "type of hearing")
   val oicHearingType: OicHearingType,
 )
+
+@Schema(description = "Outcome")
+data class OutcomeDto(
+  @Schema(description = "The id of the outcome")
+  val id: Long? = null,
+  @Schema(description = "outcome come")
```

Comment:
might be a typo - do you mean outcome code?

Reply:
yes!